edpop_explorer/readers/kb.py

```diff
@@ -54,7 +55,10 @@ def _convert_record(self, sruthirecord: dict) -> BibliographicalRecord:
             record.link = self.KB_LINK.format(record.identifier)
         record.title = self._get_title(sruthirecord)
         record.languages = self._get_languages(sruthirecord)
-        # TODO: add the other fields
+        record.extent = self._get_extent(sruthirecord)
+        record.size = self._get_size(sruthirecord)
+        record.publisher_or_printer = self._get_publisher(sruthirecord)
+        record.contributors = self._get_contributors(sruthirecord)
```

Comment:
This part of the code looks repetitive. Maybe just make `title`, `languages`, `extent` etcetera `@property`s?

Reply:
That would be good, I think, but they are part of a different class. I think this work can be done in connection of the desired refactoring mentioned in #63.